src/main/java/Duke.java

```diff
@@ -1,10 +1,225 @@
+import Exceptions.*;
+//import Tasks.Deadline;
+//import Tasks.Event;
+//import Tasks.Task;
+//import Tasks.ToDo;
+import Tasks.*;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int DEADLINE_STRING_LENGTH = 9;     // "deadline "
+    public static final int BY_SEPARATOR_LENGTH = 4;        // "/by  "
+    public static final int EVENT_STRING_LENGTH = 6;        // "event "
+    public static final int AT_SEPARATOR_LENGTH = 4;        // "/at  "
+    public static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void printList(ArrayList<Task> list) {
+            System.out.println("        ____________________________________________");
+            for (int i = 1; i <= list.size(); i += 1) {
+                String taskType = list.get(i-1).getTaskType();
+                String markDoneStatus = list.get(i-1).isDone()?"[X]":"[ ]";
+
+                switch(taskType) {
+                case "ToDo":
+                    System.out.println("        [T]" + markDoneStatus + " " + i + ". " + list.get(i-1).getTaskName());
+                    break;
+
+                case "Deadline":
+                    String deadline = list.get(i-1).getTaskDeadline();
+                    System.out.println("        [D]" + markDoneStatus + " " + i + ". " + list.get(i-1).getTaskName() +
+                            " (by: " + deadline + ")");
+                    break;
+
+                case "Event":
+                    String eventTime = list.get(i-1).getEventTime();
+                    System.out.println("        [E]" + markDoneStatus + " " + i + ". " + list.get(i-1).getTaskName() +
+                            " (at: " + eventTime + ")");
+                    break;
+
+                default:
+                    break;
+                }
+            }
+            System.out.println("        ____________________________________________");
+    }
+
+    private static void printGoodbyeMessage() {
+        System.out.println("____________________________________________");
+        System.out.println("Bye. Hope to see you again soon!");
+        System.out.println("____________________________________________");
+    }
+
+    private static void printWelcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________");
+        System.out.println("Hello! I'm Duke.");
+        System.out.println("What can I do for you?");
+        System.out.println("____________________________________________");
+    }
+
+    public static void handleInput() {
```

Comment:
Consider breaking up this long method.

https://nus-cs2113-ay2223s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods